### PR TITLE
Knative project meeting kiosk kube con eu 2023 br

### PR DESCRIPTION
**Pull request by nainaz**
"Fixes #1297" or "Knative Project meeting blog for kubeconEU23"

## Proposed Changes 

-
-
-

### CODE REVIEW

File: blog/docs/events/knative-projectmeeting-kubecon-eu-2023.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+## Knative Project Meeting on Tuesday, April 18 (Pre-conference) at 8:00 AM CET
+During this event, you'll have the opportunity to engage with Knative experts as they share their insights on the latest advancements and use cases. You'll get a chance to network with peers, ask questions, and  help define Knative's roadmap and community growth.
+Whether you're a seasoned developer or new to cloud-native computing, this meeting is the perfect opportunity to share your use cases and expectations, and expand your knowledge. Mark your calendars and join us for an informative and engaging 2 hours that get you ready to tackle your next Kubernetes project with the Serverless flair!
+P.S: This event is free for all registered attendees for the conference.
+
+## Knative Kiosk
+Knative Project will have a half day kiosk in the Expo area during the Conference.  and Kiosk is free. We are looking forward to seeing you in Amsterdam at the Project meeting and at the Knative kiosk in the expo area.
```
Comment (evankanderson) [inline blog/docs/events/knative-projectmeeting-kubecon-eu-2023.md:14]:
```suggestion
Knative Project will have a half day kiosk in the Expo area during the Conference. We are looking forward to seeing you in Amsterdam at the Project meeting and at the Knative kiosk in the expo area.
```

File: config/nav.yml [hunk 2]
```diff
@@ -304,5 +304,4 @@ nav:
         - deepc: about/case-studies/deepc.md
         - Outfit7: about/case-studies/outfit7.md
         - Puppet: about/case-studies/puppet.md
-        - PNC Bank: about/case-studies/pnc.md
-    - Blog: /blog/
```
Comment (evankanderson) [inline config/nav.yml:308]: Why does this remove the `/blog` portion of the nav here?

File: overrides/home.html [hunk 3]
```diff
@@ -314,7 +314,7 @@ <h2>Contributions Welcome</h2>
             <p class="normal-text">Want to join the fun on Github? New users are always welcome!</p>
           </a>
         </div>
-        <div class="pane">
+        <div class="col">
```
Comment (evankanderson) [inline overrides/home.html:317]: Is this accidental?

Comment (nainaz) [reply overrides/home.html:317]: Yes, I did not commit this on this branch and don't know where it is coming from. I tried rebasing and that failed too. I am getting errors could not read '.git/rebase-merge/head-name'. I have tried to continue, skip, abort, quit and all give me same error. currently, if I do log I don't see this commit but I don't know how to move this here. 
P.S: I am very new to git.

File: config/nav.yml (hunk 2 above)
Comment (nainaz) [reply config/nav.yml:308]: The file I had locally did not have it. so I think it got removed while I was merging conflict. I started working on a brand new branch and still things went haywire. I need help :(

**Pull request closed without merging**